### ISSUE TITLE
Integration tests fail on tarantool/master because of new error messages

### ISSUE THREAD

**Issue by locker** [tarantool/cartridge#1619]
A few cartridge tests [fail](https://github.com/tarantool/tarantool/runs/4374156717) when run on [tarantool/master](https://github.com/tarantool/tarantool/tree/master):

```
Failed tests:

integration.ddl.test_no_instances_to_check_schema
integration.pool_map_call.test_errors_united
integration.pool_map_call.test_negative
integration.proxy.test_dead_destination
integration.rpc.test_api
integration.rpc.test_routing
integration.slow_requests.test_netbox_timeouts
integration.switchover.stateboard.test_all_rw
integration.switchover.stateboard.test_alone_instance
integration.switchover.stateboard.test_api
integration.switchover.stateboard.test_enabling
unit.remote_control.test_late_accept
unit.remote_control.test_reconnect
unit.remote_control.test_switch_box_to_rc

Error: Process completed with exit code 14.
```

The reason of the failure is that we slightly changed net.box error messages, see https://github.com/tarantool/tarantool/pull/6653. Now, instead of `Connection refused` net.box will report something like `connect, called on fd 55, aka 127.0.0.1:34138: Connection refused`. This breaks the cartridge tests that expect an exact error message:

```
integration.ddl.test_no_instances_to_check_schema
...ad/src/tarantool/cartridge/test/integration/ddl_test.lua:270: Error message does not match pattern: "\"localhost:13301\":*Connection refused"
Error message received: "...ad/src/tarantool/cartridge/test/integration/ddl_test.lua:37: \"localhost:13301\": connect, called on fd 53, aka 127.0.0.1:56442: Connection refused"
```

Need to fix the tests to use pattern matching instead of exact string matching.

**Comment by filonenko-mikhail** [comment]
```
1) integration.join_third_instance.test_join_third_storage
407
.../tarantool/test/integration/join_third_instance_test.lua:47: There is no active coordinator
408
stack traceback:
409
	...rk/tarantool/tarantool/cartridge/test-helpers/server.lua:194: in function 'graphql'
410
	.../tarantool/test/integration/join_third_instance_test.lua:47: in function 'integration.join_third_instance.test_join_third_storage'
411
	...tool/tarantool/.rocks/share/tarantool/luatest/runner.lua:344: in function <...tool/tarantool/.rocks/share/tarantool/luatest/runner.lua:343>
412
	[C]: in function 'xpcall'
413
	...tool/tarantool/.rocks/share/tarantool/luatest/runner.lua:343: in function 'super'
414
	...l/tarantool/.rocks/share/tarantool/luatest/capturing.lua:106: in function 'protected_call'
415
	...tool/tarantool/.rocks/share/tarantool/luatest/runner.lua:420: in function 'super'
416
	...ntool/tarantool/.rocks/share/tarantool/luatest/hooks.lua:175: in function 'invoke_test_function'
417
	...tool/tarantool/.rocks/share/tarantool/luatest/runner.lua:415: in function 'super'
418
	...
419
	[C]: in function 'xpcall'
420
	...ntool/tarantool/.rocks/share/tarantool/luatest/utils.lua:37: in function 'run_tests'
421
	...tool/tarantool/.rocks/share/tarantool/luatest/runner.lua:258: in function <...tool/tarantool/.rocks/share/tarantool/luatest/runner.lua:254>
422
	[C]: in function 'xpcall'
423
	...l/tarantool/.rocks/share/tarantool/luatest/capturing.lua:74: in function <...l/tarantool/.rocks/share/tarantool/luatest/capturing.lua:72>
424
	[C]: in function 'xpcall'
425
	...tool/tarantool/.rocks/share/tarantool/luatest/runner.lua:50: in function 'fn'
426
	...tool/.rocks/share/tarantool/luatest/sandboxed_runner.lua:14: in function 'run'
427
	...antool/.rocks/share/tarantool/luatest/cli_entrypoint.lua:4: in function <...antool/.rocks/share/tarantool/luatest/cli_entrypoint.lua:3>
428
	....rocks/share/tarantool/rocks/luatest/0.5.6-1/bin/luatest:3: in main chunk
```

**Comment by filonenko-mikhail** [comment]
Tests fixed for tarantool 2.10
Every flucky tests opened in separate tasks.
